esphome/components/voice_assistant/voice_assistant.cpp

```diff
@@ -930,6 +931,63 @@ void VoiceAssistant::on_announce(const api::VoiceAssistantAnnounceRequest &msg)
 #endif
 }
 
+void VoiceAssistant::on_set_configuration(const std::vector<std::string>& active_wake_words) {
+  if (this->micro_wake_word_) {
+    // Disable all wake words first
+    for (auto &model : this->micro_wake_word_->get_wake_words()) {
+      model->disable();
+    }
+
+    // Enable only active wake words
+    for (auto ww_id_str : active_wake_words) {
+      // Wake word id is index
+      uint32_t ww_id;
+      std::stringstream ss(ww_id_str);
+      ss >> ww_id;
+
+      auto model = this->micro_wake_word_->get_wake_words().at(ww_id);
+      model->enable();
+      ESP_LOGD(TAG, "Enabled wake word: %s", model->get_wake_word().c_str());
+    }
+  }
+};
+
+const Configuration &VoiceAssistant::get_configuration() {
+  this->config_.available_wake_words.clear();
+  this->config_.active_wake_words.clear();
+
+  if (this->micro_wake_word_) {
+    this->config_.max_active_wake_words = 1;
+
+    // Wake word id is index
+    uint32_t ww_id = 0;
+    for (auto &model : this->micro_wake_word_->get_wake_words()) {
+      std::stringstream ss;
+      ss << ww_id;
+      auto ww_id_str = ss.str();
```

Comment:
We have functions for this already without bringing in the `stringstream` lib.
```suggestion
      std::string ww_id_str = to_string(ww_id);
```

Reply:
Using the model's id now